build.gradle

```diff
@@ -91,26 +92,18 @@ publishing {
             }
         }
     }
-    repositories {
-        maven {
-            if (project.hasProperty("repositoryUrl")) {
-              url = project.getProperty("repositoryUrl")
-            } else {
-              def releasesRepoUrl  = 'https://oss.sonatype.org/service/local/staging/deploy/maven2/'
-              def snapshotsRepoUrl = 'https://oss.sonatype.org/content/repositories/snapshots/'
-              url = version.endsWith('SNAPSHOT') ? snapshotsRepoUrl : releasesRepoUrl
-            }
-            if (project.hasProperty('sonatypeUsername') && project.hasProperty('sonatypePassword')) {
-                credentials {
-                    username project.property('sonatypeUsername')
-                    password project.property('sonatypePassword')
-                }
-            }
-        }
-    }
 }
 
 signing {
     required = { gradle.taskGraph.hasTask(publish) && !version.endsWith("SNAPSHOT") }
     sign publishing.publications.maven
 }
+
+nexusPublishing {
```

Comment:
I believe the gradle task for publishing should be changed from `publish` to `publishToSonatype`?
At the following line: https://github.com/gluonhq/substrate/pull/167/files#diff-6e62ddc767174617641f0b68eedf525fR9

Reply:
We can specifically make a call to `publishToSonatype`. Or, we can keep calling `publish` and it will make a call to `publishMavenJavaPublicationToSonatypeRepository` which does the same task.